apps/frontend/src/app/Data/Characters/Mika/index.tsx

```diff
@@ -121,7 +121,7 @@ const a1DetectorStacks_physical_dmg_ = equal(
   target.charKey,
   a1DetectorStacks_physical_dmg_disp
 )
-
+const [condC6CritPath, condC6Crit] = cond(key, 'c6Crit')
```

Comment:
You're not using this for buff calculation

Reply:
I don't think his C6 needs a cond actually, isn't the condition the same as inSoulWind?

Reply:
yeah, but while the regular buff snapshots, CRIT from C6 does not[citation needed] (but it shouldn't, like other crit buffs)

Reply:
oh, wait, i'm not sure if the regular buff snapshots